### PR TITLE
Fix direct access of ptif files with non-ASCII filenames.

### PR DESCRIPTION
**Pull request by manthey**
This was fixed in the pylibtiff module, but changes to that required some additional changed to our code.

### CODE REVIEW

**Comment by manthey** [comment]
@jbeezley or @cdeepakroy Can either of you review this when you get a chance?  Once it is merged in, I'll make a PR in HistomicsTK to deploy the newer version of pylibtiff.

File: .travis.yml [hunk 1]
```diff
@@ -192,7 +192,7 @@ install:
   # - npm install
   # but since there are often connection failures, use the retry package.
   - npm-install-retry
-  - BABEL_ENV=cover NYC_CWD=$main_path girder-install web --plugins=large_image,worker,jobs --dev
+  - BABEL_ENV=cover NYC_CWD=$main_path girder-install web --plugins=jobs,worker,large_image --dev
```
Comment (zachmullen) [inline .travis.yml]: This line shouldn't matter anymore :)

File: server/tilesource/tiff_reader.py [hunk 2]
```diff
@@ -268,9 +268,10 @@ def _getJpegTables(self):
         tableSize = ctypes.c_uint32()
         tableBuffer = ctypes.c_voidp()
 
-        libtiff_ctypes.libtiff.TIFFGetField.argtypes = \
-            libtiff_ctypes.libtiff.TIFFGetField.argtypes[:2] + \
-            [ctypes.POINTER(ctypes.c_uint32), ctypes.POINTER(ctypes.c_void_p)]
+        if libtiff_ctypes.libtiff.TIFFGetField.argtypes:
+            libtiff_ctypes.libtiff.TIFFGetField.argtypes = \
+                libtiff_ctypes.libtiff.TIFFGetField.argtypes[:2] + \
+                [ctypes.POINTER(ctypes.c_uint32), ctypes.POINTER(ctypes.c_void_p)]
```
Comment (zachmullen) [inline server/tilesource/tiff_reader.py]: Seems like a comment here could be very helpful

Comment (manthey) [inline server/tilesource/tiff_reader.py]: I'll add one in the future -- I responded to @jbeezley approval my merging already.

**Pull request merged**